.travis.yml

```diff
@@ -1,17 +1,32 @@
+---
```

Comment:
If you generate YAML with YAML.pm you get this start marker. You can see this in http://www.yaml.org/spec/1.2/spec.html - I think it's optional in most cases.